code/modules/clothing/glasses/glasses.dm

```diff
@@ -444,7 +435,7 @@
 	///Looping sound to play
 	var/datum/looping_sound/active_sound = /datum/looping_sound/scan_pulse
 	///How loud the looping sound should be
-	var/looping_sound_volume = 15
+	var/looping_sound_volume = 25
```

Comment:
why god, why does it even have a looping sound? Its a passive item.

Reply:
It was part of the cons of it.

Reply:
being really annoying shouldn't be a balancing factor. I'm not going to make a giga 1 shot kill gun with the draw back of breaking your eardrums.

Reply:
As I worked on the balancing of it. It isn't played only to you but others around you. To give xenos who have paid attention beforehand to it a slight heads up, it isn't ear breaking levels or shouldn't be nor the soundeffect for it to be grating. The item is about as passive as a scout cloak, you actively have to choose when to use it. It's to prevent complete blind sides without making the item completely silent or taking away another factor

Being "annoying" isn't the con of the item